sdk/include/aditof/camera.h

```diff
@@ -134,6 +134,20 @@ class SDK_API Camera {
     virtual Status requestFrame(Frame *frame,
                                 FrameUpdateCallback cb = nullptr) = 0;
 
+    /**
+     * @brief get scaled AB image with logarithmic base 10
+     * @param frame - The frame of the camera
+     * @param datasize - set with the size of the vector
+     * @param min_value_of_AB_pixel - minim value of ab image vector
+     * @param max_value_of_AB_pixel - max value of ab image vector
+     * If min_value_of_AB_pixel or max_value_of_AB_pixel is not set eg.
+     * is zero then this values are calculated from ab image vector
+     * @return a vector with scaled values
+     */
+    virtual uint8_t *getABData8bit(aditof::Frame *frame, uint32_t &datasize,
+                                   uint32_t min_value_of_AB_pixel = 0,
```

Comment:
How should figure out the min and max value of AB? Shouldn't this be detected inside the algorithm?